recipes/isoseq3/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "isoseq3" %}
 {% set version = "3.0.0" %}
-{% set sha256 = "aa6219840ed928d52114644705966c810aa0c865915d40a4eca2edc2cdadbcec" %}
+{% set sha256 = "bf43b4f7c2f7a825c954e8445b66fe3cbc6ca0d95361c11acb5e855ba19040c7" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/PacificBiosciences/IsoSeq3/releases/download/v{{ version }}/isoseq3 # [linux]
-  sha256: {{ sha256 }} # [linux]
+  url: https://github.com/PacificBiosciences/IsoSeq3/releases/download/v{{ version }}/isoseq3
```

Comment:
Just curious, this `# [linux]` comment seems to be null functional while `skip: True # [osx]` is functional. It's hard to differentiate though.

Reply:
it is functional, just unnecessary, as it causes issues with the linter when specifying a `patches:` field.